File: testlib_checker_test/ac/submission.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace submission
{
    class Program
    {
        static void Main(string[] args)
        {
            int n = int.Parse(Console.ReadLine().Trim());

            List<int> v = Console.ReadLine().Trim().Split(' ').Select(x => int.Parse(x)).ToList();

            int maxx = v[0];
            int maxxi = 0;

            for (int i = 1; i < n; i++)
            {
                if (v[i] > maxx)
                {
                    maxx = v[i];
                    maxxi = i;
                }
            }

            Console.WriteLine(maxxi);
        }
    }
}

File: testlib_checker_test/ac/submission.cpp
#include <iostream>
#include <vector>

using namespace std;

int main()
{
    int n;
    cin >> n;

    vector<int> v;
    for (int i = 0; i < n; i++)
    {
        int x;
        cin >> x;
        v.push_back(x);
    }

    int maxx = v[0];
    int maxxi = 0;

    for (int i = 1; i < n; i++)
    {
        if (v[i] > maxx)
        {
            maxx = v[i];
            maxxi = i;
        }
    }

    cout << maxxi;
    return 0;
}